package.json

```diff
@@ -29,7 +29,7 @@
   "bugs": {
     "url": "https://github.com/MetaMask/test-dapp/issues"
   },
-  "homepage": "https://github.com/MetaMask/test-dapp#readme",
+  "homepage": "https://metamask.github.io/test-dapp",
```

Comment:
Pointing to the deployed website is required for `gh-pages` to work correctly in at least some instances, and this seems like a good change regardless.